chronicles.nim

```diff
@@ -346,12 +346,12 @@ template log*(lineInfo: static InstInfo,
 template wrapSideEffects(debug: bool, body: untyped) {.inject.} =
   when debug:
     {.noSideEffect.}:
-      when (NimMajor, NimMinor) >= (1, 6):
-        {.warning[BareExcept]:off.}
+      when defined(nimHasWarnBareExcept):
+        {.push warning[BareExcept]:off.}
```

Comment:
are bareexcept warnings on or off in the `body`, with this construct?

Reply:
see also https://github.com/nim-lang/Nim/pull/21390

Reply:
Off, thought I don't expect it to be an issue in this case, unless someone does
```nim
debug hehe=try: echo "hey" expect: discard
```